Cargo.toml

```diff
@@ -18,7 +18,8 @@ exclude = [
     "proto-compiler"
 ]
 
-# [patch.crates-io]
+[patch.crates-io]
+ics23 = { git = "https://github.com/mzabaluev/ics23.git", branch = "prost-0.10" }
```

Comment:
This is the only wart that remains, pending on https://github.com/confio/ics23/pull/84 or a similar fix.

Reply:
Fixed now, but the dependency may need to be updated after pending yank: https://github.com/confio/ics23/issues/87